roles/ansible_service_broker/vars/openshift-enterprise.yml

```diff
@@ -1,9 +1,9 @@
 ---
 __ansible_service_broker_registry_type: rhcc
 __ansible_service_broker_registry_name: rh
-__ansible_service_broker_registry_url: "https://registry.redhat.io"
-__ansible_service_broker_registry_user: null
-__ansible_service_broker_registry_password: null
+__ansible_service_broker_registry_url: "{{ oreg_url | default('https://registry.redhat.io') }}"
```

Comment:
oreg_url doesn't generally include protocol and is actually a format string that likely requires some parsing, it would be something like this `registry.redhat.io/openshift3/ose-${compontent}:${version}`

Does that work here?

Reply:
No, we need just the registry URL. Thanks for pointing that out.

Maybe we should just leave this value as the original `https://registry.redhat.io` then. I'll switch it back to that, and let me know if you have any other feedback.

Reply:
good catch, no it wouldn't. Would it make sense to just hardcode this to `https://registry.redhat.io` or is there another variable that captures that piece of it without parsing?